### ISSUE TITLE
Feature extraction 

### ISSUE THREAD

**Issue by glemaitre** [glemaitre/protoclass#22]
- Pixel-based standalone
    - [ ] Intensity
    - [ ] Prewitt
    - [ ] Sobel
    - [ ] Kirsch
    - [ ] Gabor
    - [ ] Haralick
    - [ ] DCT
    - [ ] Fractal
    - [ ] Wavelet
    - [ ] Gaussian
    - [ ] Position
- Pixel-based temporal
    - [ ] Whole spectra
    - [ ] Semi-quantitative
    - [ ] Toft
    - [ ] Brix
    - [ ] Weibull
    - [ ] PUM
Region-based standalone
    - [ ] Percentile
    - [ ] Moments
    - [ ] Histograms
    - [ ] HoG
    - [ ] Shape context
    - [ ] LBP

**Comment by pkrush** [comment]
Cool. I like how your using markup here and checking stuff off.

**Comment by glemaitre** [comment]
@pkrush Yep that's pretty useful. The project feature would be even better.
